tests/app/dao/test_api_key_dao.py

```diff
@@ -64,7 +64,7 @@ def test_expire_api_key_should_update_the_api_key_and_create_history_record(noti
 
 def test_last_used_should_update_the_api_key_and_not_create_history_record(notify_api, sample_api_key):
     last_used = datetime.utcnow()
-    update_last_used_api_key(api_key_id=sample_api_key.id, last_used=last_used)
+    update_last_used_api_key(sample_api_key.id, last_used)
```

Comment:
The `rate_limit_db_calls` decorator is expecting params through `args` not `kwargs` so I adjusted the test here to not use `kwargs` instead of refactoring the decorator to check both.